pkg/fake_gcloud/lib/src/cloner.dart

```diff
@@ -0,0 +1,42 @@
+import 'dart:mirrors';
+
+/// Configures a clone helper.
```

Comment:
Needs documentation... If we need this at all, see comment on how we might not need to work at the high-level... then we don't have use mirrors and we can just rely on the mirrors already used in `gcloud`.